week-1/Homework/extra/2-piping.js

```diff
@@ -16,26 +16,41 @@
   the final result to the variable goodCode
 */
 
-function add() {
+function add(a, b) {
 
+  let adding = a + b;
+
+  // console.log(adding);
+  return  parseFloat(adding.toFixed(1));
```

Comment:
Nice!

You may be surprised this was necessary. Decimal numbers in JS (and many other languages) are expressed as "floats" which follow some rules (floating-point arithmetic) that are not always intuitive.